apps/web/app/(all)/[workspaceSlug]/(projects)/projects/(detail)/[projectId]/cycles/(list)/mobile-header.tsx

```diff
@@ -2,7 +2,7 @@
 
 import { observer } from "mobx-react";
 // ui
-import { GanttChartSquare, LayoutGrid, List } from "lucide-react";
+import { GanttChartSquare, LayoutGrid, List, type LucideIcon } from "lucide-react";
```

Comment:
import type is preferred because type thing can cause module resolution issues and end up creating references to files causing cycles

Reply:
i'll address in a follow up PR b/c this needs to be addressed in the eslint rules which will cause hundreds of more changes, too much for this PR.